components/script/dom/xmlhttprequest.rs

```diff
@@ -153,13 +153,8 @@ pub struct XMLHttpRequest {
 
 impl XMLHttpRequest {
     fn new_inherited(global: GlobalRef) -> XMLHttpRequest {
-        //TODO - update this when referrer policy implemented for workers
-        let (referrer_url, referrer_policy) = if let GlobalRef::Window(window) = global {
-            let document = window.Document();
-            (Some(document.url().clone()), document.get_referrer_policy())
-        } else {
-            (None, None)
-        };
+        let referrer_url = Some(global.get_url());
```

Comment:
Note that now the field `referrer_url` is always `Some`, so I don't think it needs to stay an option any more?